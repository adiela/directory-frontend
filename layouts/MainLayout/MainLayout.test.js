import React from 'react';
import renderer from 'react-test-renderer';
import IndexPage from "pages/index"

test('Layout displays properly', () => {
    const component = renderer.create(<IndexPage />,);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});