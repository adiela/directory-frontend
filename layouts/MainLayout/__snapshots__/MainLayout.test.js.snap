// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout displays properly 1`] = `
Array [
  <div>
    <nav>
      <div>
        Logo
      </div>
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/contact"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Contact
          </a>
        </li>
      </ul>
    </nav>
    <main>
      Home page
    </main>
  </div>,
  <footer>
    Brand name © 
    2021
  </footer>,
]
`;

exports[`Link changes the class when hovered 1`] = `
Array [
  <div>
    <nav>
      <div>
        Logo
      </div>
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/contact"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Contact
          </a>
        </li>
      </ul>
    </nav>
    <main>
      Home page
    </main>
  </div>,
  <footer>
    Brand name © 
    2021
  </footer>,
]
`;
